code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -284,7 +284,7 @@
 	return 1
 
 /datum/species/proc/handle_temperature(datum/gas_mixture/breath, var/mob/living/carbon/human/H) // called by human/life, handles temperatures
-	if( (abs(310.15 - breath.temperature) > 50) && !(RESIST_HEAT in H.mutations)) // Hot air hurts :(
+	if(abs(310.15 - breath.temperature) > 50) // Hot air hurts :(
```

Comment:
check here: https://github.com/ParadiseSS13/Paradise/pull/4209/files#diff-2de2df982739a4fbdb54a4b2715d6f92R475
here: https://github.com/ParadiseSS13/Paradise/pull/4209/files#diff-2de2df982739a4fbdb54a4b2715d6f92R451
and here: https://github.com/ParadiseSS13/Paradise/pull/4209/files#diff-2de2df982739a4fbdb54a4b2715d6f92R556

These 3 confer total fire immunity, with the exception of what you breathe in. If you even have internals, you're immune to 100% of the fire's effects.